tests/foreman/cli/test_capsule_installer.py

```diff
@@ -203,11 +202,11 @@ def test_positive_reinstall_on_same_node_after_remove(self):
             result = capsule_vm.run('yum install -y satellite-capsule')
             self.assertEqual(result.return_code, 0)
             # generate capsule certs and installer command
-            cert_file_path = '/tmp/{0}-certs.tar'.format(capsule_vm.hostname)
+            cert_file_path = f'/tmp/{capsule_vm.hostname}-certs.tar'
             result = ssh.command(
                 'capsule-certs-generate '
-                '--foreman-proxy-fqdn {0} '
-                '--certs-tar {1}'.format(capsule_vm.hostname, cert_file_path)
+                '--foreman-proxy-fqdn {} '
+                '--certs-tar {}'.format(capsule_vm.hostname, cert_file_path)
```

Comment:
i think this reads better
```suggestion
                f'--foreman-proxy-fqdn {capsule_vm.hostname} '
                f'--certs-tar {cert_file_path}'
```